massa-models/src/node_configuration/default_testing.rs

```diff
@@ -92,7 +92,7 @@ pub const POS_LOOKBACK_CYCLES: u64 = 2;
 pub const ROLL_PRICE: Amount = Amount::from_raw(100 * AMOUNT_DECIMAL_FACTOR);
 pub const SLOT_KEY_SIZE: usize = 9;
 pub const THREAD_COUNT: u8 = 2;
-pub const T0: MassaTime = MassaTime::from(32000);
+pub const T0: MassaTime = MassaTime::from(32000); // sometimes overriden in config.rs or setting.rs
```

Comment:
I'd rather used 1000 as default in tests

Reply:
Not really the purpose of the PR, I'll test later to change that :wink: